cmd/create.go

```diff
@@ -78,13 +80,21 @@ func newCreateCmd() *cobra.Command {
 }
 
 func (cc *createCmd) initConfig() error {
+	if cc.subDir != "" {
+		log.Debug("updating destination")
+		cleanPath, err := filematches.GetSubDirPath(cc.dest, cc.subDir)
+		if err != nil {
+			return err
+		}
+		cc.dest = cleanPath
```

Comment:
what is the objective of the new subdirectory flag? It looks like it currently functions as an alias to the destination flag. Is it doing something different than calling `draft create -d dir` vs `draft create -s dir`? I may have missed the if there is a spot where it intentionally does not execute this substitution in a way specific to subdirectories

Reply:
From what I was told this is to make Draft able to handle repos with multiple projects easier. Letting it use the -s flag for the subdirectories makes it easier to consume it as a product more than it helps with people directly using it in the command line.

Reply:
I will look at the way I set it up though because you make a good point about it basically being an alias.